### PR TITLE
 Screpyy96 finished connect section

### PR DESCRIPTION
**Pull request by screpyy96**

### CODE REVIEW

File: web/css/styles.css [hunk 1]
```diff
@@ -67,6 +83,34 @@ a[class^="disc-link__"] {
 .disc-link__granny-apple {
   background-color: #c1f8c0;
 }
+.disc-link__oasis {
+  background-color: #fdebcf;
+}
+
+.disc-link__onahau-light {
+  background-color: #ccf5ff;
+}
```
Comment (RalucaRbc) [inline web/css/styles.css:92]: You need an empty line between the divs 😄

File: web/css/styles.css [hunk 2]
```diff
@@ -67,6 +83,34 @@ a[class^="disc-link__"] {
...
+.disc-link__blue-chalk {
+  background-color: #e3d6e9;
+}
```
Comment (RalucaRbc) [inline web/css/styles.css:95]: And here

File: web/css/styles.css [hunk 3]
```diff
@@ -67,6 +83,34 @@ a[class^="disc-link__"] {
...
+.disc-link__pastel-pink {
+  background-color: #ffd1dc;
+}
```
Comment (RalucaRbc) [inline web/css/styles.css:98]: And here 😆

File: web/css/styles.css [hunk 4]
```diff
@@ -67,6 +83,34 @@ a[class^="disc-link__"] {
...
+.disc-link__moon-raker {
+  background-color: #dfddf9;
+}
+.disc-link__janna {
+  background-color: #f5ead9;
+}
+.disc-link__white-ice {
+  background-color: #e1e7e4;
+}
```
Comment (RalucaRbc) [inline web/css/styles.css:110]: And so on 😄

**Review by RalucaRbc — APPROVED** [review]
Looks good so far, few readability things

**Review by robertoalx — APPROVED** [review]
💯

**Review by psyhedeliq — APPROVED** [review]
LGTM! 🚀

**Review by Constantin-Matei — APPROVED** [review]

**Review by biggicode — APPROVED** [review]

**Comment by MariaTeo** [comment]
Keep it up 😋

**Pull request merged**